src/commands/deploy/pipeline.ts

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (c) 2021, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import { Messages } from '@salesforce/core';
+import { SfCommand } from '@salesforce/sf-plugins-core';
+import { PromotePipelineResult, validateTestFlags } from '../../common';
+import {
+  branchName,
+  bundleVersionName,
+  deployAll,
+  devopsCenterProjectName,
+  devopsCenterUsername,
+  specificTests,
+  testLevel,
+} from '../../common/flags';
+
+Messages.importMessagesDirectory(__dirname);
+const messages = Messages.loadMessages('@salesforce/plugin-devops-center', 'deploy.pipeline');
+
+export default class DeployPipeline extends SfCommand<PromotePipelineResult> {
```

Comment:
Just a small detail here, in [W-11991482](https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07EE00001BF8ADYA1/view) we name this class Pipeline.

Reply:
I thought it would be better to have a more specific class name.

Reply:
:nice: